content/teams_intro.tex

```diff
@@ -87,13 +89,12 @@ \subsubsection*{Team Creation}
 contexts (see Section~\ref{sec:ctx}) or using the translation routine
 \FUNC{shmem\_team\_translate\_pe}.
 
-As with any collective routine on a team, the program must ensure that there
+As with any collective routine on a team, the program must ensure that in multithreaded executions there
```

Comment:
executions**,**

Reply:
@wrrobin I've rewritten this sentence in 9eb5db1 to reference the full description of team-based collective requirements and to also hopefully make it a little easier to digest. Please let me know what you think.

Reply:
@jdinan The updated text looks good. Thanks.